src/zino/scheduler.py

```diff
@@ -74,17 +79,17 @@ def init_state_for_devices(devices: Sequence[PollDevice]):
 
 
 async def load_and_schedule_polldevs(polldevs_conf: str):
-    new_devices, deleted_devices = load_polldevs(polldevs_conf)
-    schedule_new_devices(new_devices)
-    deschedule_deleted_devices(deleted_devices)
+    new_devices, deleted_devices, changed_devices = load_polldevs(polldevs_conf)
+    deschedule_deleted_devices(deleted_devices | changed_devices)
+    schedule_new_devices(new_devices | changed_devices)
```

Comment:
What do you think about renaming these functions now that they are used for more than just new and deleted devices?
Renaming to `deschedule_devices` and `schedule_devices` might be better now. Then we can change the log message again to just be `_log.debug("Scheduling %s devices", len(devices))`

Reply:
I think that is a very good idea